app/models/pin.rb

```diff
@@ -0,0 +1,10 @@
+class Pin < ApplicationRecord
+  validates :description, presence: true
+  validates :image_url, presence: true
+  validates :article_url, presence: true
+  validates :board, presence: true
+
```

Comment:
As per our conversation earlier @wlffann @akintner , we decided to leave this as is.